deployment/Deploy.ps1

```diff
@@ -227,6 +227,9 @@ try {
     $ConnectUpdateOp = Update-LtiFunctionAppSettings $resourceGroupName $deploymentOutput.properties.outputs.ConnectFunctionName.value $EdnaKeyString
     $PlatformsUpdateOp = Update-LtiFunctionAppSettings $resourceGroupName $deploymentOutput.properties.outputs.PlatformsFunctionName.value $EdnaKeyString
     $UsersUpdateOp = Update-LtiFunctionAppSettings $resourceGroupName $deploymentOutput.properties.outputs.UsersFunctionName.value $EdnaKeyString
+    $AssignmentsUpdateOp = Update-LtiFunctionAppSettings $resourceGroupName $deploymentOutput.properties.outputs.AssignmentsFunctionName.value $EdnaKeyString
```

Comment:
I think we could consider using `forEach` to improve the verbosity of this code. But that would require us to add a small change in `azuredeploy.json` which helps return all the functions as an container in `outputs`.

Reply:
Maybe we can consider modifying `outputs` in `azuredeploy.json` to have all the function apps listed as a subobject. This way we would iterate over all the function apps to add `EdnaKeyString` appsetting on each of them.